### ISSUE TITLE
[Bug]: failed to create mysql database: [400] [replication_type] Configuring replication_type is no longer supported; [engine] invalid engine

### ISSUE THREAD

**Issue by maxLeet** [linode/terraform-provider-linode#1669]
### Terraform Version

terraform v1.3.9 on darwin_arm64 

### Linode Provider Version

registry.terraform.io/linode/linode v2.31.0

### Effected Terraform Resources

linode_database_mysql
linode_database_postgresql

### Terraform Config Files

resource "linode_database_mysql" "foobar" {
  label = "mydatabase"
  engine_id = "mysql/8.0.30"
  region = "us-southeast"
  type = "g6-nanode-1"
}

### Debug Output

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with
the following symbols:
  + create

Terraform will perform the following actions:

  # linode_database_mysql.foobar will be created
  + resource "linode_database_mysql" "foobar" {
      + allow_list       = (known after apply)
      + ca_cert          = (sensitive value)
      + cluster_size     = 1
      + created          = (known after apply)
      + encrypted        = false
      + engine           = (known after apply)
      + engine_id        = "mysql/8.0.30"
      + host_primary     = (known after apply)
      + host_secondary   = (known after apply)
      + id               = (known after apply)
      + label            = "mydatabase"
      + region           = "us-southeast"
      + replication_type = "none"
      + root_password    = (sensitive value)
      + root_username    = (sensitive value)
      + ssl_connection   = false
      + status           = (known after apply)
      + type             = "g6-nanode-1"
      + updated          = (known after apply)
      + version          = (known after apply)

      + updates {
          + day_of_week   = (known after apply)
          + duration      = (known after apply)
          + frequency     = (known after apply)
          + hour_of_day   = (known after apply)
          + week_of_month = (known after apply)
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes     

linode_database_mysql.foobar: Creating...
╷
│ Error: failed to create mysql database: [400] [replication_type] Configuring replication_type is no longer supported; [engine] invalid engine
│ 
│   with linode_database_mysql.foobar,
│   on linode_terrform_mysql.tf line 14, in resource "linode_database_mysql" "foobar":
│   14: resource "linode_database_mysql" "foobar" {
│ 
╵

### Expected Behavior

Expected behavior is to create a mysql database node on the appropriate linode account using the configuration and resources provided in the terraform file. 

### Actual Behavior

 Error: failed to create mysql database: [400] [replication_type] Configuring replication_type is no longer supported; [engine] invalid engine
 
 replication_type is not used in the configuration file.

### Steps to Reproduce

Create a database using the the resource provided by terraform using linode as the provider.

**Comment by maxLeet** [comment]
Updated terraform from v1.3.9 to v1.9.8 issue is still persisting.

**Comment by lgarber-akamai** [comment]
Hey @maxLeet, thanks for the report!

This is a known incompatibility between our new managed databases platform and our Terraform provider. You can read our release notes for more information: https://techdocs.akamai.com/cloud-computing/changelog/nov-14-2024-database

We're working towards restoring compatibility and will let you know once we have an update to share.

**Comment by maxLeet** [comment]
Thank you!

Looking forward to the update.

**Comment by adelowo** [comment]
> Hey @maxLeet, thanks for the report!
> 
> This is a known incompatibility between our new managed databases platform and our Terraform provider. You can read our release notes for more information: https://techdocs.akamai.com/cloud-computing/changelog/nov-14-2024-database
> 
> We're working towards restoring compatibility and will let you know once we have an update to share.

is there a known timeline?

**Comment by sizzlorox** [comment]
Currently have a project waiting on this fix, would be nice to have a timeline.

**Comment by DavidTheProgrammer** [comment]
Just bumped into this today too. Though it was my token but error persisted even with full rights. The comment above was informative, a timeline would be even better.

**Comment by godartm** [comment]
Currently have a project waiting on this fix too.
Would be nice to have a timeline.

**Comment by maxLeet** [comment]
Any updates on this issue? I received a noticed because it has been 30days without any activity and I would like to continue using linode for my terraform implementation.

**Comment by lgarber-akamai** [comment]
Sorry for the delay on responding to this! 
Databases can now be provisioned using the [linode_database_mysql_v2](https://registry.terraform.io/providers/linode/linode/latest/docs/resources/database_mysql_v2) and [linode_database_postgresql_v2](https://registry.terraform.io/providers/linode/linode/latest/docs/resources/database_postgresql_v2) resources
